tests/untldoc_test.py

```diff
@@ -756,3 +756,9 @@ def test_generate_hash():
     test_input = [[('qualifier', 'serialtitle'), ('content', 'The Bronco')]]
     hash_val = untldoc.generate_hash(test_input)
     assert hash_val == '9eff715f7ee7da9d5c2efdf075d07225'
+
+
+def test_get_record_version():
+    untl_elements = untldoc.untlxml2py('tests/metadc_complete.untl.xml')
+    version_hash = untldoc.get_record_version(untl_elements)
+    assert version_hash == 'ac83772c3b4ecb0bc925dca4f7793a0f'
```

Comment:
I'm really not sure if we _should_ do this, but something that comes to mind in order to test the intended use of this new addition is to maybe within this same test then modify the untl_elements dict to change one of the "unmeaningful" meta values, assert the hash remains the same, then change some "meaningful" meta value and assert that the hash does change. Maybe that's overkill though? I'm not sure. Just a thought.

Reply:
I think for it's future intended use, we should add more test. I think I may separate them instead of have them all in one test. So, I will add 2 more tests.